src/poetry/packages/locker.py

```diff
@@ -82,6 +83,7 @@ def is_fresh(self) -> bool:
         metadata = lock.get("metadata", {})
 
         if "content-hash" in metadata:
+            print(self._content_hash, lock["metadata"]["content-hash"])
```

Comment:
Was this just for testing and can be cleaned up?

Reply:
ugg; yeah. fixed.